src/main/java/ui/components/pickers/PickerMilestone.java

```diff
@@ -0,0 +1,179 @@
+package ui.components.pickers;
+
+import backend.resource.TurboMilestone;
+import com.sun.javafx.tk.FontLoader;
+import com.sun.javafx.tk.Toolkit;
+import javafx.scene.Node;
+import javafx.scene.control.Label;
+import javafx.scene.text.Font;
+
+public class PickerMilestone extends TurboMilestone implements Comparable<PickerMilestone> {
+    public static final String OPEN_COLOUR = "#84BE54";
+    public static final String CLOSE_COLOUR = "#AD3E27";
+    private static final int SMALL_LABEL_FONT = 12;
+    private static final int BIG_LABEL_FONT = 16;
+    boolean isSelected = false;
+    boolean isHighlighted = false;
+    boolean isFaded = false;
+    boolean isExisting = false;
+
+    public PickerMilestone(TurboMilestone milestone) {
+        super(milestone.getRepoId(), milestone.getId(), milestone.getTitle());
+        setDueDate(milestone.getDueDate());
+        setDescription(milestone.getDescription() == null ? "" : milestone.getDescription());
```

Comment:
Can this ever happen? TurboMilestone is supposed to have eliminated null by this point.